### ISSUE TITLE
[IsarInspector] _InternalLinkedhashMap<String,dynamic> is not subtype of Map<String,Object> in type cast

### ISSUE THREAD

**Issue by frogman1189** [isar/isar#217]
**Steps to Reproduce**
Using m1 Mac, running MacOS 11.6
- Start project with inspector enabled
- Load Quotes
- open Isar Inspector 2.1.4
- paste isar inspector url in
- go to a page which has data
- Error occurs:
![Screen Shot 2022-02-03 at 11 09 20 AM](https://user-images.githubusercontent.com/20494957/152245803-f8c1085c-c2b8-47ea-9f70-e652db6bb80a.png)

Worth noting, version 1.0.5 works, but I haven't been able to get anything above that to function

**Code sample**
using sample code from https://github.com/isar/samples

updated Pubspec to use 2.1.4:
```
...
dependencies:
  flutter:
    sdk: flutter
  isar: ^2.1.4
  isar_flutter_libs: ^2.1.4
  lazy_load_scrollview: 1.3.0
  auto_size_text: ^3.0.0
  flutter_staggered_grid_view: 0.4.0-nullsafety.3
  path_provider: ^2.0.8 

dev_dependencies:
  isar_generator: ^2.1.4
  build_runner: any
...
```

updated main to use embedded Isar inspector
```
void main() async {
  WidgetsFlutterBinding.ensureInitialized();
  final isar = await Isar.open(
    inspector: true,
    schemas: [QuoteSchema],
    directory: (await getApplicationDocumentsDirectory()).path,
  );
  runApp(QuotesApp(
    isar: isar,
  ));
}
...
```

**Version**
 - Platform: IOS (quotes), Mac (Isar Inspector)
 - Flutter version: 2.8.1
 - Isar version: 2.1.4
 - Isar Generator version: 2.1.4

**Comment by red-star25** [comment]
What's the solution to this problem?

**Comment by Faaatman** [comment]
@red-star25 the developer merged fix #274  a couple of days ago and I guess will be releasing a new version with the fix soon. we just have to be patient :).

**Comment by dev-thinks** [comment]
@leisim looks like the fix for this issue has been merged. looking forward to get a release. thank you.

**Comment by tijanirf** [comment]
> dev-thinks

Just to confirm, so the fix is not included in version 2.2.1 right?

**Comment by Faaatman** [comment]
> Just to confirm, so the fix is not included in version 2.2.1 right?

Correct, it isn't included in version 2.2.1.
